tests/unit/lms/services/lti_h_test.py

```diff
@@ -70,16 +70,6 @@ def test_sync_calls_bulk_action_correctly(self, h_api, h_user, lti_h_svc):
             CommandBuilder.group_membership.create("user_0", "group_1").raw,
         ]
 
-    def test_sync_raises_if_theres_no_ApplicationInstance(
```

Comment:
This is not handled in the code so it was just testing python exception propagation.

No application instance would have failed before reaching this code so removing this test.